lib/private/Authentication/Token/PublicKeyToken.php

```diff
@@ -191,7 +187,7 @@ public function getScopeAsArray(): array {
 		return $scope;
 	}
 
-	public function setScope($scope) {
+	public function setScope(mixed $scope): void {
```

Comment:
Could be more specific on the supported types:
```suggestion
	public function setScope(string|array $scope): void {
```

Reply:
It can also be null it seems